data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/type/MultipartCatalogDescriptionRequestSender.java

```diff
@@ -79,16 +78,16 @@ public Message buildMessageHeader(CatalogRequest request, DynamicAttributeToken
                 ._recipientConnector_(Collections.singletonList(URI.create(request.getConnectorId())))
                 .build();
         //TODO: IDS REFACTORING: incorporate this into the protocol itself
-        message.setProperty(Range.FROM, request.getRange().getFrom());
-        message.setProperty(Range.TO, request.getRange().getTo());
+        message.setProperty(CatalogRequest.RANGE, request.getRange());
```

Comment:
see my comment in `DescriptionRequestHandler` regarding transmission of separate fields vs. a serialized `QuerySpec`

Reply:
Done.